deliveries/Archisinal-milestone_1.md

```diff
@@ -0,0 +1,41 @@
+
+**Deliverables**
+
+| Number | Deliverable | Link                                                                                   | Notes                                                                                                                                                                                        |                             
+| ------------- | ------------- |----------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 0a. | License | [LICENSE](https://github.com/archisinal/marketplace-contracts/blob/main/LICENSE)                   | MIT                                                                                                                                                                                          |
+| 0b. | Documentation | [README.md](https://github.com/archisinal/marketplace-contracts/blob/main/README.md)                      | Entry-point of docs is README.md, then use `cargo doc --open` to check all docs.                                                                                                             |
+| 0c. | Testing and Testing Guide | [TESTING.md](https://github.com/archisinal/marketplace-contracts/blob/main/TESTING.md)             | Created E2E, Security, Performance tests, and testing guide for them.                                                                                                                        |
+| 0d. | Docker | [Dockerfile](https://github.com/archisinal/marketplace-contracts/blob/main/Dockerfile), [CI](https://github.com/Archisinal/marketplace-contracts/blob/main/.github/workflows/ci.yml)             | Created dockerfile for testing, also CI for automatic checks.                                                                                                                                |
+| 0e. | Article | [Medium: Archisinal Technologies — Genesis Scoping & MVP](https://medium.com/p/3df888e1de98)                                                                            |                                                                                                                                                                                              |
+| 1. | NFT Minting | [ArchNFT](https://github.com/archisinal/marketplace-contracts/tree/main/contracts/arch_nft)        | Implemented in ArchNFT contract (PSP34 logic from OpenBrush). Also ArchNFT contains different metadata such as collection royalty.                                                                                                                                                                                             |
```

Comment:
This is for all the contracts: Code could generally use more comments around public members

Reply:
Valid comment for open source code. In the [commit](https://github.com/Archisinal/marketplace-contracts/commit/34053460d9e9c4d041dbd782d627b66099118e28) I added more line comments across the codebase. We’ll continue sticking to this pattern.

Reply:
I was specifically referring to _public members_ here. So structs, enums, etc. marked as `pub`. This is much more important to do well than inline comments since this is what people on docs.rs (or editor hints) will be using as a reference.

Inline comments can sometime be unhelpful, for [example here](https://github.com/Archisinal/marketplace-contracts/commit/34053460d9e9c4d041dbd782d627b66099118e28#diff-48ea9993142697540aa23eeab03a45297ba3eb348e27ca7ff30323fdb8cded65R80):

<img width="496" alt="image" src="https://github.com/use-inkubator/Grant-Milestone-Delivery/assets/12344623/35258575-25a7-4704-96bd-a837ff389a99">

This doesn't give the reader any more info than they would've gotten by reading the code.